src/helpers/assetSelectors.js

```diff
@@ -65,14 +66,20 @@ const sortAssetsByNativeAmount = (
     noValue = EMPTY_ARRAY,
   } = groupAssetsByMarketValue(assetsNativePrices);
 
-  const sortedAssetsNoShitcoins = sortList(
-    hasValue,
-    'native.balance.amount',
-    'desc',
-    0,
-    toNumber
-  );
-  const sortedShitcoins = sortList(noValue, 'name', 'asc');
+  const sortedAssetsNoShitcoins = hasValue.sort((a, b) => {
+    let itemA = Number(a.native?.balance?.amount) ?? 0;
+    let itemB = Number(b.native?.balance?.amount) ?? 0;
+
+    return itemA < itemB ? 1 : -1;
+  });
+
+  const sortedShitcoins = noValue.sort((a, b) => {
+    let itemA = a.name;
+    let itemB = b.name;
+
+    return itemA > itemB ? 1 : -1;
+  });
```

Comment:
I have always had a problem with "pretending" that this is immutable when this is not.